kubernetes-data-platform/values.yaml

```diff
@@ -34,8 +34,10 @@ presto:
       rollingUpdate:
         maxSurge: 1
         maxUnavailable: 1
+    deployPrometheusExporter: false
+    useJmxExporter: false
   ingress:
-    enable: true
+    enable: false
```

Comment:
if we're targeting local development with our "sane defaults" does it make more sense to have this set to true and let the users turn it off in prod for security reasons?

Reply:
Can minikube do ingresses? Where do we get the needed annotations?

Reply:
yes; we will need to default to running one of the other non-AWS specific ingress controllers and then flip over to the one specific to your cloud provider via chart deploy-time config. i will create a separate card for that one.